easybuild/easyconfigs/l/LAMMPS/LAMMPS-2Aug2023-foss-2023a-kokkos.eb

```diff
@@ -0,0 +1,167 @@
+name = 'LAMMPS'
+version = '2Aug2023'
+versionsuffix = '-kokkos'
+
+homepage = 'https://www.lammps.org'
+description = """LAMMPS is a classical molecular dynamics code, and an acronym
+for Large-scale Atomic/Molecular Massively Parallel Simulator. LAMMPS has
+potentials for solid-state materials (metals, semiconductors) and soft matter
+(biomolecules, polymers) and coarse-grained or mesoscopic systems. It can be
+used to model atoms or, more generically, as a parallel particle simulator at
+the atomic, meso, or continuum scale. LAMMPS runs on single processors or in
+parallel using message-passing techniques and a spatial-decomposition of the
+simulation domain. The code is designed to be easy to modify or extend with new
+functionality.
+"""
+
+toolchain = {'name': 'foss', 'version': '2023a'}
+toolchainopts = {'openmp': True, 'usempi': True}
+
+# 'https://github.com/lammps/lammps/archive/'
+source_urls = [GITHUB_LOWER_SOURCE]
+sources = ['stable_%(version)s_update2.tar.gz']
```

Comment:
In the past we haven't reflected the update in the version number, but we probably should?

PR #19396 uses `-update1-kokkos`, should we adopt something similar here?

In addition, I'm in favor of using a more sensible versioning scheme like `20230802`, but that may surprise end users who are looking for `2Aug2023`?

Thoughts on this @ocaisa ?

Reply:
I think you've got to match what LAMMPS actually uses